scanner.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/python3
+import nmap

```

Comment:
goal of port scanning module is to provide a extremely fast overview of top 1K ports which it already performs properly, everyone will use nmap in a later stage anyways, so this is just an extra dependency not serving much purpose